Examples/ExampleMultiProjectIntegration/.safedi/configuration/include.csv

```diff
@@ -0,0 +1 @@
+Subproject
```

Comment:
`Examples/ExampleMultiProjectIntegration/ExampleMultiProjectIntegration` is already scanned because it's the root module. We just need to list the paths to the other module(s) in this file.